source/community/faq/index.md

```diff
@@ -47,6 +51,18 @@ IIIF allows for:
  * [Authentication API][auth]
  * In Progress: [IIIF for Audio/Visual materials][av]
 
+##  How does IIIF relate to my current, internal interoperability layer?
+
+* Some institutions implement IIIF using a shim, or translation layer, on top of their existing interoperability layer.  In other cases, IIIF is implemented without dependencies on existing services. IIIF provides for different levels of compliance, beginning with the IIIF Image API. For more information, please see [IIIF technical details][tech-details].  
+
+## Does IIIF replace or integrate with my Digital Asset Management (DAM) software?
+
+* IIIF is a method for delivery of assets, not storage. Integration with your DAM software is dependent on your DAM provider. The IIIF community is currently in the midst of asking popular DAMS providers to support IIIF.
+
+## Is IIIF an aggregator?
+
+* No, IIIF does not aggregate content.  IIIF defines APIs that allow for interoperability between digital image repositories.  Because these APIs are well documented and understood, use of IIIF can enable aggregators such as DPLA and Europeana to access your images more easily.  
```

Comment:
Here IIIF == community, I think?

No, the IIIF community does not have a single aggregation platform for content. ... image repositories, including aggregators.  

The rationale here is that ArtStor is an aggregator by value (the actual images) and makes IIIF APIs available.